h2o-algos/src/main/java/hex/generic/GenericModelOutput.java

```diff
@@ -39,8 +39,11 @@ public GenericModelOutput(final ModelDescriptor modelDescriptor, final ModelAttr
             _cross_validation_metrics_summary = convertTable(modelAttributes.getCrossValidationMetricsSummary());
 
             if (modelAttributes instanceof SharedTreeModelAttributes) {
-                fillSharedTreeModelAttributes((SharedTreeModelAttributes) modelAttributes, modelDescriptor);
-            } else {
+                fillSharedTreeModelAttributes(((SharedTreeModelAttributes) modelAttributes).getVariableImportances(), modelDescriptor);
+            } else if (modelAttributes instanceof  DeepLearningModelAttributes){
+                fillSharedTreeModelAttributes(((DeepLearningModelAttributes) modelAttributes).getVariableImportances(), modelDescriptor);
```

Comment:
maybe the method `fillSharedTreeModelAttributes ` should be renamed (since it works for DL algo as well)?

Reply:
+1 also the name of the method does not really represent what the method does

Reply:
Done ! Thank you.

Reply:
also the two if branches are identical and could be merged into one with || operator

Reply:
Actually, the method was

- incorrectly named
- one parameter was unused

so I got rid of it completely and only use `convertVariableImportances` method directly.

Reply:
There is explicit type cast, each model might have different attributes and once it's `SharedTreeModelAttributes`, second time it's `DeepLearningModelAttributes`. Those do not have anything in common.